source/manuals/programming-languages/js.html.md.erb

```diff
@@ -110,14 +110,6 @@ Use soft tabs with a two space indent.
   sets the appropriate affordances. It also follows the conventions of the
   standard library.
 
-## CoffeeScript
```

Comment:
I guess this was in here because it's still the [default thing in Rails](https://guides.rubyonrails.org/working_with_javascript_in_rails.html). Could we replace it with a generalised section on using compiles-to-js languages? (For example, what do we think about TypeScript?)

Reply:
Hmm maybe we just leave the section in I wasnt aware it's still a default.